[DOCS] Create New Landing Pages

kwcanuck opened the pull request:
## Description
Create new landing pages for docs TOC.

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

kwcanuck committed:
Preliminary landing page set up

ghost commented:
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=7993&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/great-expectations/great_expectations/7993/9f896a90/b759b9d84299eb1b46cc3c5d1d7bc3e9dd8857da.svg)](https://app.codesee.io/r/reviews?pr=7993&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
<img src="https://app.codesee.io/Legend_v3.png" alt="CodeSee Map legend" width="220" height="156">
</details>

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'landing-pages' of https://github.com/great-expectations/great_expectations into landing-pages

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Link fix.

kwcanuck committed:
Added color list to CSS

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'landing-pages' of https://github.com/great-expectations/great_expectations into landing-pages

kwcanuck committed:
Image test

kwcanuck committed:
Image update.

kwcanuck committed:
Image update.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Image update.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Image update.

kwcanuck committed:
Image update.

kwcanuck committed:
Image update.

kwcanuck committed:
Image update.

kwcanuck committed:
Image update.

kwcanuck committed:
Image update.

kwcanuck committed:
Image update

kwcanuck committed:
Image update.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

HaebichanGX committed:
Check simple spelling and word order error using ChatGPT-based program

Check simple spelling and word order error for all published docs using ChatGPT-based program.

kwcanuck committed:
Merge branch 'develop' into chatgpt_doc_validator_all_doc_check

kwcanuck committed:
Apply suggestions from code review

kwcanuck committed:
Merge branch 'develop' into chatgpt_doc_validator_all_doc_check

kwcanuck committed:
Update docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage.md

kwcanuck committed:
Merge branches 'landing-pages' and 'chatgpt_doc_validator_all_doc_check' of https://github.com/great-expectations/great_expectations into landing-pages

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'landing-pages' of https://github.com/great-expectations/great_expectations into landing-pages

kwcanuck committed:
Update custom.scss

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit

kwcanuck committed:
Minor edit.

Rachel-Reverie committed:
gets cloud storage icon to appear

Rachel-Reverie committed:
Merge remote-tracking branch 'origin/landing-pages' into landing-pages

# Conflicts:
#	docs/docusaurus/docs/guides/setup/setup_overview_tiles.md

Rachel-Reverie committed:
relocates icon to static folder

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Topic consolidation

kwcanuck committed:
Topic consolidation

kwcanuck committed:
Added redirects. Minor edits.

kwcanuck committed:
TOC edits

kwcanuck committed:
Minor edits.

kwcanuck committed:
Topic consolidation.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Topic consolidation

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Topic consolidation.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Topic consolidation.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Topic consolidation.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Topic consolidation

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit

kwcanuck committed:
Update sidebars.js

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Reorg.

kwcanuck committed:
Minor edit

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Updated link.

kwcanuck committed:
MInor edit.

kwcanuck committed:
Revert "MInor edit."

This reverts commit eafe0f53a5b384fe6c82fadf870cd81f9150ed5b.

kwcanuck committed:
Revert "Updated link."

This reverts commit eb2b2e70adabe1625de5e2afc62b389c6b7d08ab.

kwcanuck committed:
Revert "Minor edit."

This reverts commit 46527dc8d53f087febc64d3fdd9d6e884863bbc5.

kwcanuck committed:
Update connect_filesystem_source_data.md

kwcanuck committed:
Update connect_filesystem_source_data.md

kwcanuck committed:
Reorg.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Topic consolidation.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Topic consolidation

kwcanuck committed:
Topic consolidation.

kwcanuck committed:
Minor edits

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edits.

kwcanuck committed:
Landing page edits

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Landing page edits

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits

kwcanuck committed:
Minor edit

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Revert "Minor edit"

This reverts commit bea7be8b3678d6576ffb8adf60e8e521f51753db.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Added landing pages minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Landing page edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits

kwcanuck committed:
Landing page edits.

kwcanuck committed:
Landing page edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Revert "Minor edits."

This reverts commit b80184d410d704ad322af3f286f6694317f97dce.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Added landing pages for vaildate data and minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Validate Data lp updates.

kwcanuck committed:
Added Create Expectations lp

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Edited Create and manage Expectations lp

kwcanuck committed:
Minor edits.

kwcanuck committed:
Profilers and Data Assistants lp edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits and toc reorg.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Updated Reference landing page.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edits.

kwcanuck committed:
Added auto-generate to TOC.

kwcanuck committed:
Auto-generated landing page edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Added icons.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits

kwcanuck committed:
Minor edit.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edits.

kwcanuck committed:
Update aws_lp.md

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
TOC update.

kwcanuck committed:
Revert "TOC update."

This reverts commit cf3176028ddc1f362ec98a80be52909d455d6676.

kwcanuck committed:
Minor TOC edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
TOC edits.

kwcanuck committed:
TOC edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edits.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edit.

kwcanuck committed:
Minor edits.

Rachel-Reverie commented on docs/docusaurus/docs/guides/expectations/add_features_custom_expectations_lp.md:
```diff
@@ -0,0 +1,19 @@
+  <LinkCard topIcon label="Add comments to Expectations and display them in Data Docs" description="Add descriptive comments to Expectations and display them in Data Docs" href="/docs/guides/expectations/advanced/how_to_add_comments_to_expectations_and_display_them_in_data_docs" icon="/img/comment_icon.svg" />
+  <LinkCard topIcon label="Create example cases for a Custom Expectation" description="Add example cases to document and test the behavior of your Expectation" href="/docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation" icon="/img/example_cases_icon.svg" />
+  <LinkCard topIcon label="Add input validation and type checking to a Custom Expectation" description="Add validation and Type Checking to the input parameters of a Custom Expectation" href="/docs/guides/expectations/features_custom_expectations/how_to_add_input_validation_for_an_expectation" icon="/img/validation_check_icon.svg" />
+  <LinkCard topIcon label="Add Spark support for Custom Expectations" description="Add native Spark support for your Custom Expectations" href="/docs/guides/expectations/features_custom_expectations/how_to_add_spark_support_for_an_expectation" icon="/img/spark_icon.svg" />
```
Is there an official spark logo for this card?  I think we use one for the integrations page.

kwcanuck replied:
@Rachel-Reverie I replaced the existing logo (which was an official logo I manipulated).

Rachel-Reverie commented on docs/docusaurus/docs/guides/expectations/add_features_custom_expectations_lp.md:
```diff
@@ -0,0 +1,19 @@
+  <LinkCard topIcon label="Create example cases for a Custom Expectation" description="Add example cases to document and test the behavior of your Expectation" href="/docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation" icon="/img/example_cases_icon.svg" />
+  <LinkCard topIcon label="Add input validation and type checking to a Custom Expectation" description="Add validation and Type Checking to the input parameters of a Custom Expectation" href="/docs/guides/expectations/features_custom_expectations/how_to_add_input_validation_for_an_expectation" icon="/img/validation_check_icon.svg" />
+  <LinkCard topIcon label="Add Spark support for Custom Expectations" description="Add native Spark support for your Custom Expectations" href="/docs/guides/expectations/features_custom_expectations/how_to_add_spark_support_for_an_expectation" icon="/img/spark_icon.svg" />
+  <LinkCard topIcon label="Add SQLAlchemy support for Custom Expectations" description="Add native SQLAlchemy support for your Custom Expectations" href="/docs/guides/expectations/features_custom_expectations/how_to_add_sqlalchemy_support_for_an_expectation" icon="/img/alchemy_icon.svg" />
```
Does sqlalchemy have an official logo we should use for this card?

kwcanuck replied:
@Rachel-Reverie the logo has been added as discussed.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Added edits requested after peer review.

kwcanuck committed:
Minor edit.

Rachel-Reverie reviewed (approved):
Looks good to me.  I really like a lot of the changes.

I'm approving so you're covered for sidebar.js code owner.  You'll need to get an okay on the icons before actually merging.

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Merge branch 'develop' into landing-pages

kwcanuck committed:
Minor edit.
